webpack.config.js

```diff
@@ -69,8 +70,20 @@ module.exports = {
                 exclude: /node_modules/,
                 use: "babel-loader",
             },
+            {
+                test: /\.scss$/,
+                use: ExtractTextPlugin.extract( {
```

Comment:
Yes, all the examples that I have seen have the options declared in module -> rules. Including the show cases the ExtractTextPlugin repository has on github.